reports/README.md

```diff
@@ -0,0 +1,10 @@
+## Authoring krabby reports
+
+all krabby reports should follow the [unix philosophy](http://en.wikipedia.org/wiki/Unix_philosophy), "Do One Thing and Do It Well". It should only be looking at a single source of information (whether that be the filesystem, an API, or otherwise) and processing the data on a single metric. It should be as simple and easy as possible for the end user to use multiple reports to accomplish the "big picture" they are looking for.
+
+#### getting going
+
+Each report is a function that is given two arguments, `results`, and `callback`.
+
+`results` is an array of every `test` that was ran. You will want to iterate result to process it. You can read more about the object signature of each test [here](https://github.com/patrickkettner/krabby/tree/master/tests).
```

Comment:
I actually don't understand what you mean here? it will be an array with all the result from previous tests?